lib/python/gladevcp/tooledit_widget.py

```diff
@@ -539,8 +661,9 @@ def main(filename=None):
     window.vbox.add(tooledit)
     window.connect("destroy", gtk.main_quit)
     tooledit.set_col_visible("abcUVW", False, tab='1')
-    #tooledit.set_filename("/home/chris/emc2-dev/configs/sim/gscreen/gscreen_custom/lathe-fanucy.tbl")
-    tooledit.set_filename("/home/chris/emc2-dev/configs/sim/lathe.tbl")
+    # uncommented the below line for testing.
+    tooledit.set_filename("/home/jim/linuxcnc/configs/sim.gmoccapy/tool.tbl")
```

Comment:
I see that even before your change, the "tooledit.set_filename" call was not commented out, but does this work properly for end users who do not have a file named `/home/jim/linuxcnc/configs/sim.gmoccapy/tool.tbl`?